templates/stamp/utilities.sh

```diff
@@ -95,8 +97,9 @@ configure_datadisks()
     # installation cannot be made silent using the techniques that keep the
     # mdadm installation quiet: a) -y AND b) DEBIAN_FRONTEND=noninteractive.
     # Therefore, we'll install postfix early with the "No configuration" option.
-    echo "postfix postfix/main_mailer_type select No configuration" | sudo debconf-set-selections
-    sudo apt-get install -y postfix
```

Comment:
fyi, don't need sudo tags here because the script is always run as elevated (preconditions exist in both callers)